src/cc/libbpf.c

```diff
@@ -177,9 +187,14 @@ int bpf_prog_load(enum bpf_prog_type prog_type,
   if (ret < 0 && !log_buf) {
     // caller did not specify log_buf but failure should be printed,
     // so call recursively and print the result to stderr
+    buffer_size = attr.insn_cnt * 128;
```

Comment:
I'd rather not have such formula propagated everywhere and copy pasted by other projects. The size of verifier log_buf has no correlation to program size.